manifests/apis/webhooks/kustomizeconfig.yaml

```diff
@@ -0,0 +1,28 @@
+nameReference:
+- kind: Issuer
+  group: cert-manager.io
+  fieldSpecs:
+  - kind: Certificate
+    group: cert-manager.io
+    path: spec/issuerRef/name
+
+- kind: Service
+  version: v1
+  fieldSpecs:
+  - kind: ValidatingWebhookConfiguration
+    group: admissionregistration.k8s.io
+    path: webhooks/clientConfig/service/name
```

Comment:
[Name references](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/transformerconfigs/README.md#name-reference-transformer) tie a target resource's name to a list of other resources' referrers' names. Once tied, the referrers' names will change alongside the target name via transformers like namePrefix and nameSuffix.

So the first item in this section says "`spec.issuerRef.name` references the name of a `cert-manager.io Issuer`." This enables kustomize to apply the same transformation to both fields such that a change to an Issuer's name is _also_ applied to a Certificates `spec.issuerRef.name`.